qmetaobject/src/qtdeclarative.rs

```diff
@@ -231,6 +231,54 @@ impl QmlEngine {
         })
     }
 
+    pub fn invoke_method_noreturn(&mut self, name: QByteArray, args: &[QVariant]) -> QVariant {
```

Comment:
If the method doesn't return,  why is it returning a `-> QVariant` ?

Reply:
Yup, my bad. I'll fix it a bit later.

By the way `QMetaObject::invokeMethod` itself returns `true` [" if the member could be invoked"](https://doc.qt.io/qt-5/qmetaobject.html#invokeMethod). Maybe it is makes sense to return `Result<QVariant>` in `invoke_method` and `Result<()>` in `invoke_method_noreturn`?

Reply:
Fixed!

As for a `invoke_method` signature change - I think I will open another PR some time later.